src/openms/include/OpenMS/FORMAT/FileHandler.h

```diff
@@ -139,14 +141,24 @@ namespace OpenMS
     /// Non-mutable access to the options for loading/storing
     const PeakFileOptions& getOptions() const;
 
+    /// Mutable access to the feature file options for loading/storing
+    FeatureFileOptions& getFeatOptions();
```

Comment:
One of the reasons why splitting FileHandler into FeatureIO PeakIO would be better ;)

Reply:
Yeah. It's gonna happen. The problem is I'm trying to keep the steps as separate as possible. This is already a lot.